docs/pages/pmd/devdocs/pmd_devdocs_metrics_howto.md

```diff
@@ -0,0 +1,16 @@
+---
+title:  Using code metrics in custom rules
+tags: [customizing]
+summary: "PMD was recently enhanced with the ability to compute code metrics on Java code (the so-called Metrics 
+Framework). This framework provides developers with a straightforward interface to use code metrics in their rules, and 
+to extend the framework with their own custom metrics."
+last_updated: July 20, 2017
+sidebar: pmd_sidebar
+permalink: pmd_devdocs_metrics_howto.html
```

Comment:
Okay I'll rename it. 

I think it would be nice to have a proper path in urls, it's more structured and overall cleaner. I fiddled a bit with jekyll but found no way to do that... The documentation theme's doc seems to [hint that this is not possible](http://idratherbewriting.com/documentation-theme-jekyll/mydoc_pages.html#where-to-save-pages):

>The listing of all pages in the root directory (which happens when you add a permalink property to the pages) is what allows the relative linking and offline viewing of the site to work.

Besides the sidebar seems to remove any forward slash it finds in urls, so we cannot even specify urls manually. I guess the theme doesn't give us the choice